core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java

```diff
@@ -86,6 +86,15 @@
   boolean promoteBlock(final long blockId) throws IOException, AlluxioException;
 
   /**
+   * Removes a block.
+   *
+   * @param blockId The id of the block that will be removed
```

Comment:
since you are doing cleanup, could you fix up style in this file (s/The/the)? thanks